.eslintrc.js

```diff
@@ -5,6 +5,11 @@ module.exports = {
     node: true,
   },
   parser: 'vue-eslint-parser',
+  parserOptions: {
+    parser: 'espree',
+    ecmaVersion: 2022,
+    sourceType: 'module',
+  },
```

Comment:
The pre-2022 parser gets very confused by asyncs/awaits appearing in the `<script setup>` tag